### DISCUSSION TITLE
Displaying unit for product weight when editing product

### DISCUSSION THREAD

**Discussion by piloujp** [zencart/zencart#6232] [Ideas]
In product edit page, you don't know which unit is set for weight when filling the weight field. It might help, especially if you are not alone to add products, to display the weight unit near the field.
![Capture_unit](https://github.com/zencart/zencart/assets/117885587/6b5e5c81-9190-4223-bd0f-cd9ba99bba13)

It is very easy to do, just add this code, ` . ' (' . TEXT_PRODUCT_WEIGHT_UNIT . ')'` after `TEXT_PRODUCTS_WEIGHT` in all four collect_info.php files in admin. Full line change would be like this:
`      <?php echo zen_draw_label(TEXT_PRODUCTS_WEIGHT . ' (' . TEXT_PRODUCT_WEIGHT_UNIT . ')', 'products_weight', 'class="col-sm-3 control-label"'); ?>`

**Comment by drbyte** [comment]
Proposed: https://github.com/zencart/zencart/pull/6233

**Comment by piloujp** [comment]
Great! I am more than happy with this proposition! Japanese Language pack will benefit greatly from that (in Japan, most of the shipping method use dimension).
